spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  describe 'initialize' do
+    it 'creates a new facility factory' do
+      factory = FacilityFactory.new
+      expect(factory).to be_an_instance_of(FacilityFactory)
+    end
+  end
+
+  describe '#create_facilities(state_abbv)' do
+    it 'creates DMVs specific to the specified state' do
+      factory = FacilityFactory.new
+      facilities = factory.create_facilities("CO")
+      facilities.each do |facility|
+      expect(facility).to be_an_instance_of(Facility)
+      expect(facility.address).to include("CO")
+      expect(factory.create_facilities("HI")).to eq("sorry, we don't yet serve HI")
+      end
```

Comment:
(This applies to several `do`/`end` blocks in your submission)
Be sure to keep an eye on your indentation:
```
facilities.each do |facility|
  expect(facility).to be_an_instance_of(Facility)
  ...
end
```